pyproject.toml

```diff
@@ -78,11 +79,21 @@ build-backend = "poetry.core.masonry.api"
 remove-all-unused-imports = true
 ignore-init-module-imports = true
 ignore-pass-after-docstring = true
+in-place = true
+recursive = true
```

Comment:
an example for pre-commit hook args for autoflake `args: [ --remove-all-unused-imports, --ignore-init-module-imports, --ignore-pass-after-docstring, --in-place ]`